nautobot_golden_config/models.py

```diff
@@ -129,10 +131,36 @@ def _verify_get_custom_compliance_data(compliance_details):
             raise ValidationError(VALIDATION_MSG.format(val, "String or Json", compliance_details[val]))
 
 
+def _get_hierconfig_remediation(obj):
```

Comment:
Probably this method has to be moved to helpers/utils.py or netutils - @itdependsnetworks @jeffkala  ?